pkgs/servers/http/nginx/generic.nix

```diff
@@ -116,6 +116,14 @@ stdenv.mkDerivation {
       '';
     })
     ./nix-skip-check-logs-path.patch
+    (fetchpatch {
```

Comment:
Why bother with fetchpatch at all? The file is a plain text file that just applies as patch. It doesn't matter much but `fetchpatch` shouldn't be required in this case.

Reply:
Because it needs the `extraPrefix` special casing, that is only in `fetchpatch`.

Reply:
Oh! D'oh!

Reply:
Isn't it needed to prefix the paths in this case?

Reply:
Yep.

Reply:
:roll_eyes: github being slow to sync today...